sight_service/worklist_scheduler_opt.py

```diff
@@ -121,9 +128,9 @@ def GetOutcome(
   @overrides
   def decision_point(
       self, request: service_pb2.DecisionPointRequest
-    ) -> service_pb2.DecisionPointResponse:
+  ) -> service_pb2.DecisionPointResponse:
     method_name = "decision_point"
-    logging.debug(">>>>  In %s of %s", method_name, _file_name)
+    logging.info(">>>>  In %s of %s", method_name, _file_name)
```

Comment:
Meet suggested we use logging.debug. Please sync up with him on the preferred logging level.

Reply:
yes , updated to info just for testing purposes , as cloud logging not logging .debug